Core/include/Acts/Seeding/SeedFinderConfig.hpp

```diff
@@ -25,111 +25,147 @@ template <typename SpacePoint>
+  // If useVariableMiddleSPRange is true, the radial range will be calculated
+  // based on the maximum and minimum r values of the space-points in the event
+  // and a deltaR (deltaRMiddleMinSPRange, deltaRMiddleMaxSPRange)
   float deltaRMiddleMinSPRange = 10. * Acts::UnitConstants::mm;
   float deltaRMiddleMaxSPRange = 10. * Acts::UnitConstants::mm;
-  // range defined in vector for each z region
-  std::vector<std::vector<float>> rRangeMiddleSP;
-  // range defined by rMinMiddle and rMaxMiddle
-  float rMinMiddle = 60.f * Acts::UnitConstants::mm;
-  float rMaxMiddle = 120.f * Acts::UnitConstants::mm;
 
-  // z of last layers to avoid iterations
+  // Vector containing minimum and maximum z boundaries for cutting middle
+  // space-points
   std::pair<float, float> zOutermostLayers{-2700 * Acts::UnitConstants::mm,
                                            2700 * Acts::UnitConstants::mm};
 
-  // cut to the maximum value of delta z between SPs
-  float deltaZMax =
-      std::numeric_limits<float>::infinity() * Acts::UnitConstants::mm;
+  /*
+   * Seeding parameters used to define the cuts on space-point doublets
+   */
 
-  // enable cut on the compatibility between interaction point and SPs
-  bool interactionPointCut = false;
+  // Minimum radial distance between two doublet components (prefere
+  // deltaRMinTopSP and deltaRMinBottomSP to set separate values for outer and
+  // inner space-points)
+  float deltaRMin = 5 * Acts::UnitConstants::mm;
+  // Maximum radial distance between two doublet components (prefere
+  // deltaRMaxTopSP and deltaRMacBottomSP to set separate values for outer and
+  // inner space-points)
+  float deltaRMax = 270 * Acts::UnitConstants::mm;
+  // Minimum radial distance between middle-outer doublet components
+  float deltaRMinTopSP = std::numeric_limits<float>::quiet_NaN();
+  // Maximum radial distance between middle-outer doublet components
+  float deltaRMaxTopSP = std::numeric_limits<float>::quiet_NaN();
+  // Minimum radial distance between inner-middle doublet components
+  float deltaRMinBottomSP = std::numeric_limits<float>::quiet_NaN();
+  // Maximum radial distance between inner-middle doublet components
+  float deltaRMaxBottomSP = std::numeric_limits<float>::quiet_NaN();
 
-  // non equidistant binning in z
-  std::vector<float> zBinEdges;
+  // Maximum value of z-distance between space-points in doublet
+  float deltaZMax =
+      std::numeric_limits<float>::infinity() * Acts::UnitConstants::mm;
 
-  // seed confirmation
-  bool seedConfirmation = false;
-  // parameters for central seed confirmation
-  SeedConfirmationRangeConfig centralSeedConfirmationRange;
-  // parameters for forward seed confirmation
-  SeedConfirmationRangeConfig forwardSeedConfirmationRange;
+  // Maximum allowed cotTheta between two space-points in doublet, used to check
+  // if forward angle is within bounds
+  float cotThetaMax = 10.01788;  // equivalent to 3 eta (pseudorapidity)
```

Comment:
```suggestion
  float cotThetaMax = 10.01788;  // equivalent to eta = 3 (pseudorapidity)
```
(Only if possible without linebreak)